.github/workflows/e2e-tests.yml

```diff
@@ -133,9 +133,9 @@ jobs:
           key: ${{ steps.jetpack-build-cache.outputs.cache-primary-key }}
 
   e2e-tests:
-    name: "${{ matrix.project }} e2e tests"
+    name: '${{ matrix.project }} e2e tests'
```

Comment:
Same comment as above.

Reply:
Never mind, I'm wrong! :^D

Reply:
Heh, what's going on here? I didn't touch these, was it the prettify doing its thing?

Reply:
Also possible :) I have no idea how some of my editing modes are set up.